libs/components/tabs/src/lib/modules/tabs/tabset-nav-button.component.html

```diff
@@ -2,7 +2,13 @@
   *ngIf="isVisible"
   class="sky-btn sky-margin-inline-sm"
   [attr.aria-controls]="ariaControls"
-  [disabled]="disabled"
+  [disabled]="
+    disabled !== undefined
```

Comment:
The logic for this was in the getter previously; however, it was nasty. I don't love having this logic in the template though - so we can chat if needed.

Reply:
This is a good candidate for a pure pipe.

Reply:
I think I did this right. Please let me know if not.